include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -1103,6 +1103,20 @@ struct OrtApi {
    * and that's recommended because turning this option on may hurt model accuracy.
    */
   ORT_API2_STATUS(SetGlobalDenormalAsZero, _Inout_ OrtThreadingOptions* tp_options);
+
+  /**
+  * Use this API to create the configuration of an arena that can eventually be used to define 
+  * an arena based allocator's behavior
+  * max_mem : use 0 to allow ORT to choose the default
+  * arena_extend_strategy :  use -1 to allow ORT to choose the default, 0 = kNextPowerOfTwo, 1 = kSameAsRequested
```

Comment:
>arena_extend_strategy  [](start = 4, length = 22)

I think we agreed to use Doxygen style with \param or @param

Reply:
Fixed